README.md

```diff
@@ -22,7 +22,7 @@ You will need the following things properly installed on your computer.
 ## Running / Development
 
 * `ember serve` - Uses dummy API responses provided by mirage
-* `ember serve --environment=devlocal` - Sends api calls to `localhost:3000`, assumes you are running [labs-zap-api](https://github.com/NYCPlanning/labs-zap-api) locally.
+* `HOST_API=http://localhost:3000 ember serve --environment=devlocal` - Sends api calls to `localhost:3000`, assumes you are running [labs-zap-api](https://github.com/NYCPlanning/labs-zap-api) locally.
```

Comment:
We could simplify this a little more by having another environment variable of our own:

`HOST_API=http://localhost:3000 MIRAGE_ENABLED=false ember serve`

Take a look at how this one does: https://github.com/adlius/ember-osf-web/blob/develop/config/environment.js#L306-L308

I can set this up if you prefer, let me know

Reply:
Thank you! I'll put it in the backlog, and see if I can pick it up later